app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Api::V1::CommentsController < ApplicationController
```

Comment:
- [ ] Please you should follow the steps in this [article](https://guides.rubyonrails.org/api_app.html#changing-an-existing-application) to change your application to an api:- 
1- you should change your application controller class to be ::API instead of ::Base 
2- you should add this line to your config/application.rb 
`config.api_only = true
`
3- you are using an api now how about deleting these lines from the application controller or you can add a new application controller file in your api directory
  `before_action :authenticate_user?`
  `before_action :update_allowed_parameters, if: :devise_controller? `
so the user can use the api without signing in
 look how your project work after doing these changing
![image](https://user-images.githubusercontent.com/74744787/218269560-c8e9053c-bd55-4c9a-8394-e763dc906304.png)